src/domain/core/src/messages/core_message_types.rs

```diff
@@ -48,6 +55,7 @@ impl Message for DatasetLifecycleMessage {}
 pub struct DatasetLifecycleMessageCreated {
     pub dataset_id: DatasetID,
     pub owner_account_id: AccountID,
+    pub dataset_visibility: DatasetVisibility,
```

Comment:
Not critical, but we need to have a general solution for migration.
There is a database with these events already in Europort environment.
At this stage we can still manually reset some tables, but in general we would need a migration path.
In this particular case, it might live with some reasonable deserialization default.